### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by sm2017** [james-alex/date_time_format#3]
How can I add new locales?

**Comment by james-alex** [comment]
Are you looking to add new timezone identifiers in the [timezones.dart](https://github.com/james-alex/date_time_format/blob/master/lib/src/helpers/timezones.dart) file as used by the [_timeZoneAbbreviation] method in [date_time_format.dart](https://github.com/james-alex/date_time_format/blob/master/lib/src/date_time_format.dart)?

**Comment by sm2017** [comment]
No, currently I want to localize the output of `relative` method

```
// 5 minutes
print(DateTimeFormat.relative(dateTime.subtract(Duration(minutes: 5))));
```

**Comment by james-alex** [comment]
I'm currently writing an update to add support for localization for every locale supported by the [intl](https://pub.dev/packages/intl) package.

It may take me up to a few days to complete. I'll let you know when it's been published.

**Comment by freddydrodev** [comment]
any update on this

**Comment by JerryLatham203** [comment]
I am having difficulty gathering the correct time for the US time zone when adding a  utc_offset to the DateTime method, although it would be the correct time using a foreign time zone like in Europe or Africa. This occurs vice versa as well, does anyone know what I would need to input in my code to correct this? And I cannot use a subtracted and added parse method at the same time.
